easybuild/easyblocks/o/opencv.py

```diff
@@ -90,9 +91,6 @@ def prepare_step(self, *args, **kwargs):
     def configure_step(self):
         """Custom configuration procedure for OpenCV."""
 
-        if 'CMAKE_BUILD_TYPE' not in self.cfg['configopts']:
```

Comment:
Shouldn't we add this check whether `CMAKE_BUILD_TYPE` is already specified` in `configopts` in `CMakeMake`?

Reply:
Not required. This is added first so configopts can overwrite it

Reply:
OK